src/main/host/descriptor/socket/unix.rs

```diff
@@ -21,7 +21,7 @@ use crate::utility::event_queue::{EventQueue, Handle};
 use crate::utility::stream_len::StreamLen;
 use crate::utility::HostTreePointer;
 
-const UNIX_SOCKET_DEFAULT_BUFFER_SIZE: usize = 212_992;
+const UNIX_SOCKET_DEFAULT_BUFFER_SIZE: u64 = 212_992;
```

Comment:
This was changed to u64 since the socket's send limits are u64. Linux uses an int for the send buffer size, so I may change this to u32 in a future PR.